Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -167,24 +168,28 @@ class BocList
       }
     });
   
-    $(selectorControl).click(function (evt)
+    selectorControl.addEventListener('click', function (evt)
     {
       evt.stopPropagation();
       BocList.OnRowSelectorClick();
-      $(row).trigger ('click');
+      row.dispatchEvent (new MouseEvent('click'));
     });
   
       // Enable the entire selector control's cell for click events that mimic the selector control.
-    $(cell).click(function (evt)
+    cell.addEventListener('click', function (evt)
     {
       BocList.OnRowSelectorClick();
     });
   
     // Enable the entire row to the left of the selector control's cell for click events that mimic the selector control.
-    $(cell).prevAll().click(function (evt)
+    let prev: Nullable<Node> = cell;
+    while (prev = prev.previousSibling)
     {
-      BocList.OnRowSelectorClick();
-    });
+      prev.addEventListener('click', function (evt)
+      {
+        BocList.OnRowSelectorClick();
+      });
+    }
```

Comment:
please update to a for-loop. We try to avoid having two effects per statement, here it's assignment and test for null